diff --git a/rules/BUILD b/rules/BUILD
index c8b565d..b21b536 100644
--- a/rules/BUILD
+++ b/rules/BUILD
@@ -118,6 +118,21 @@ bzl_library(
     ],
 )
 
+# Publicly-visible single-file `rules.bzl` target, for users of
+# `bazelbuild/bazel-skylib`'s `bazel_skylib_gazelle_plugin`:
+# https://registry.bazel.build/modules/bazel_skylib_gazelle_plugin
+bzl_library(
+    name = "rules",
+    srcs = ["rules.bzl"],
+    visibility = [
+        "//visibility:public",
+    ],
+    deps = [
+        ":bzl",
+    ],
+)
+
+
 bzl_library(
     name = "min_sdk_version_bzl",
     srcs = ["min_sdk_version.bzl"],
